<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeWidget</class>
 <widget class="QWidget" name="ModeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModeWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="6">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageMode">
      <widget class="QPushButton" name="pushButtonMachine">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>101</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/new/prefix1/image/machine.png</normaloff>:/new/prefix1/image/machine.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>人机对战</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>170</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>网络对战</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonNetwork">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>101</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/new/prefix1/image/network.png</normaloff>:/new/prefix1/image/network.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageMachine">
      <widget class="QPushButton" name="pushButtonBlack">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/new/prefix1/image/black.png</normaloff>:/new/prefix1/image/black.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>黑子</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonWhite">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>60</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/new/prefix1/image/white.png</normaloff>:/new/prefix1/image/white.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>150</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>白子</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageNetwork">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>301</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>对方端口:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPort">
          <property name="text">
           <string>8888</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QRadioButton" name="radioButtonBlack">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>黑子</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonWhite">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>130</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>白子</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButtonSure">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>170</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/image/buttonBK.png);</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>301</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>对方ip:  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditIP">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/new/prefix1/image/close.png</normaloff>:/new/prefix1/image/close.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请选择对战模式</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>131</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>183</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonBack">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/new/prefix1/image/back.png</normaloff>:/new/prefix1/image/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
